ecs-init/docker/docker.go

```diff
@@ -32,6 +34,21 @@ const (
 	defaultDockerEndpoint = "/var/run"
 	// networkMode specifies the networkmode to create the agent container
 	networkMode = "host"
+	// minBackoffDuration specifies the minimum backoff duration for ping to
+	// return a success response from the docker socket
+	minBackoffDuration = time.Second
+	// maxBackoffDuration specifies the maximum backoff duration for ping to
+	// return a success response from docker socket
+	maxBackoffDuration = 5 * time.Second
+	// backoffJitterMultiple specifies the backoff jitter multiplier
+	// coefficient when pinging the docker socket
+	backoffJitterMultiple = 0.2
+	// backoffMultiple specifies the backoff multiplier coefficient when
+	// pinging the docker socket
+	backoffMultiple = 2
+	// maxRetries specifies the maximum number of retries for ping to return
+	// a successful response from the docker socket
+	maxRetries = 4
```

Comment:
I don't have any data on how long it should be.  That's why I'm wondering.  If you think 12s is long enough for retry I'm okay with that.